<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>maniSimulator</class>
 <widget class="QMainWindow" name="maniSimulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>maniSimulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="autobutton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>auto</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="manualbutton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>manual</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearbutton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x position</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>530</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y position</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operatebutton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>operate</string>
    </property>
   </widget>
   <widget class="QSlider" name="joint1">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>330</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="joint2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>370</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="joint3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>410</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>350</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>joint1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>390</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>joint2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>430</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>joint3</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="link1">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>55</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="link2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>95</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="link3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>135</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>link1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>100</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>link2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>140</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>link3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="joint1_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>345</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="joint2_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>385</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="joint3_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>425</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmbutton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>170</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>confirm</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="X_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="Y_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>691</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>947</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumani_simul">
    <property name="title">
     <string>mani_simul</string>
    </property>
   </widget>
   <addaction name="menumani_simul"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
